equinox-shipping/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,91 @@
+module FinalizationTransaction
+
+module Events =
+
+    let [<Literal>] CategoryId = "FinalizationTransaction"
+    let streamName clientId = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | FinalizationRequested of {| containerId : string; shipmentIds : string[] |}
+
+        | AssignmentCompleted   of {| containerId : string; shipmentIds : string[] |}
```

Comment:
completed should not bear a `containerId`

Reply:
The `containerId` is required in `AssignmentCompleted` event as the next action is to `FinalizeContainer` and the `containerId` needs to be provided.

Reply:
The State and Action need it - the Event does not

Reply:
Nice catch, I've removed the `containerId` from the events while maintaining it in the state. Thanks!

Reply:
you da man - re-reviewing now...